tests/core/fsck-on-boot/task.yaml

```diff
@@ -35,6 +35,7 @@ execute: |
       fi
       umount /var/lib/snapd/seed
       umount /run/mnt/ubuntu-seed
+      umount /boot/efi
```

Comment:
Ideally this would only be unmounted if it is mounted, like /run/mnt/snapd since this mount point won't exist on ARM systems when the test is run there, but perhaps @sergiocazzolato can take care of that in the other PR which is updating this test to work on ARM.